src/app.js

```diff
@@ -139,10 +130,14 @@ const slotBuilder = (slot) => {
     head.append(utils.dom('span', {}, {}, name));
     const list = ulBuilder(params);
 
-    if (document.getElementById(slotHelper.getDomId(slot)) != null) {
-        const slotEl = document.getElementById(slotHelper.getDomId(slot));
+    const slotEl = document.getElementById(slotHelper.getDomId(slot));
+    //Outputting 'slotEl' to aid in debugging for verifying correct selection
+    console.log('slotElements:', slotEl);
```

Comment:
console.log should probably not be in prod code

Reply:
@mabramishvili i left it on purpose to debug in the future all elements on a page, but sure i can remove
<img width="575" alt="Screenshot 2024-05-15 at 2 15 35 PM" src="https://github.com/forbes/ad-inspector/assets/60854679/ba7c47d3-fd84-40d9-aa0e-b6a947972b0c">

Reply:
removed